fwdctl/src/dump.rs

```diff
@@ -32,7 +42,12 @@ pub(super) async fn run(opts: &Options) -> Result<(), api::Error> {
     let db = Db::new(opts.db.clone(), &cfg.build()).await?;
     let latest_hash = db.root_hash().await?;
     let latest_rev = db.revision(latest_hash).await?;
-    let mut stream = latest_rev.stream();
+    let start_key_bytes = opts.start_key.as_bytes();
+    let start_key = match start_key_bytes.is_empty() {
```

Comment:
All this work will just happen in the args, you'll just have to do:
`let start_key = opts.start_key.clone().unwrap_or(Box::new([]));`

Or something like that. You could use the other iterator by mapping instead of starting with an empty slice, with a little more work:
```
    let iter = opts
        .start_key
        .as_ref()
        .map(|key| latest_rev.stream_from(key.clone()))
        .unwrap_or(latest_rev.stream());
```